applications/composer/frontend/src/Pages/SentenceDetails.tsx

```diff
@@ -132,10 +132,50 @@ const SentencesDetails = () => {
     setIsLoading(true);
     const transition = sentence?.available_transitions[selectedIndex];
     sentenceService
-      .doTransition(sentence, transition)
-      .then((sentence: Sentence) => fetchNextSentence(sentence));
+      .doTransition(sentence, transition);
   };
-  
+
+  const clickNextSentence = () => {
+    const fetchNextSentence = async (sentence: Sentence) => {
```

Comment:
question: fetchNextSentence is now repeated right? can we just have it defined once?

Reply:
cleaning it up already, give me 5 min